hoomd/hpmc/pair/lennard_jones.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2009-2023 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Lennard Jones pair potential.
+
+.. invisible-code-block: python
+
+    simulation = hoomd.util.make_example_simulation()
+    sphere = hoomd.hpmc.integrate.Sphere()
+    sphere.shape['A'] = dict(diameter=0.0)
+    simulation.operations.integrator = sphere
+
+    pair =  hoomd.hpmc.pair.LennardJones()
+    pair.params[('A', 'A')] = dict(epsilon=1, sigma=1, r_cut=2.5)
+
+    logger = hoomd.logging.Logger()
+"""
+
+import hoomd
+
+from .pair import Pair
+
+
+class LennardJones(Pair):
+    """Lennard-Jones pair potential (HPMC).
+
+    Args:
+        default_r_cut (float): Default cutoff radius :math:`[\\mathrm{length}]`.
+        default_r_on (float): Default XPLOR on radius
+          :math:`[\\mathrm{length}]`.
+        mode (str): Energy shifting/smoothing mode.
+
+    `LennardJones` computes the Lennard-Jones pair potential between every pair
+    of particles in the simulation state. The functional form of the potential,
+    including its behavior under shifting modes, is identical to that in
+    the MD pair potential `hoomd.md.pair.LJ`.
+
+    See Also:
+        `hoomd.md.pair.LJ`
+
+        `hoomd.md.pair`
+
+    .. rubric:: Example
+
+    .. code-block:: python
+
+        lennard_jones =  hoomd.hpmc.pair.LennardJones()
+        lennard_jones.params[('A', 'A')] = dict(epsilon=1, sigma=1, r_cut=2.5)
+        simulation.operations.integrator.pair_potentials = [lennard_jones]
+
+    .. py:attribute:: params
+
+        The potential parameters. The dictionary has the following keys:
+
+        * ``epsilon`` (`float`, **required**) -
+          Energy well depth :math:`\\varepsilon` :math:`[\\mathrm{energy}]`.
+        * ``sigma`` (`float`, **required**) -
+          Particle size :math:`\\sigma` :math:`[\\mathrm{length}]`.
```

Comment:
```suggestion
          Characteristic length scale :math:`\\sigma` :math:`[\\mathrm{length}]`.
```
Should we be more exact and say that sigma isn't strictly a particle size?

Reply:
Maybe say "effective particle size"?

Reply:
"Characteristic length scale" is the most accurate name. We should update the MD documentation to match.